cmd/kor/root.go

```diff
@@ -60,6 +60,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&opts.NoInteractive, "no-interactive", false, "Do not prompt for confirmation when deleting resources. Be careful using this flag!")
 	rootCmd.PersistentFlags().BoolVarP(&opts.Verbose, "verbose", "v", false, "Verbose output (print empty namespaces)")
 	rootCmd.PersistentFlags().StringVar(&opts.GroupBy, "group-by", "namespace", "Group output by (namespace, resource)")
+	rootCmd.PersistentFlags().BoolVarP(&opts.PrintReason, "print-reason", "r", false, "Print reason resource is considered unused")
```

Comment:
I suggest renaming `--print-reason` flag to `--show-reason` in similarity to kubectl flags.